jumpscale/packages/vdc_dashboard/bottle/api/deployments.py

```diff
@@ -336,23 +334,24 @@ def update():
         branch = branch_param
     else:
         branch = os.environ.get("SDK_VERSION", "development")
-    cmds = [f"git checkout {branch}", "git pull"]
-    for cmd in cmds:
-        rc, out, err = j.sals.process.execute(cmd, cwd="/sandbox/code/github/threefoldtech/js-sdk")
-        if rc:
-            return HTTPResponse(
-                j.data.serializers.json.dumps(
-                    {"error": "failed to pull upstream", "stderr": err, "stdout": out, "code": rc, "cmd": cmd,}
-                ),
-                status=500,
-                headers={"Content-Type": "application/json"},
-            )
+    # cmds = [f"git checkout {branch}", "git pull"]
+    # for cmd in cmds:
+    cmd = f"bash /sandbox/code/github/threefoldtech/js-sdk/jumpscale/packages/vdc_dashboard/scripts/update.sh {branch}"
```

Comment:
I think we can use the relative path so if we change the repo dir in the future, change in one place `cwd`. what do you think?
```suggestion
    cmd = f"bash jumpscale/packages/vdc_dashboard/scripts/update.sh {branch}"
```

Reply:
indeed much better.
we also could make use of env var PWD, it is set on the server already to `/sandbox/code/github/threefoldtech/js-sdk` what do you think?